packaging/debian/control.base

```diff
@@ -5,7 +5,7 @@ Priority: optional
 Architecture: all
 Essential: no
 Maintainer: github.com/Shopify/shopify-app-cli
-Depends: ruby (>= 2.3)
+Depends: ruby (>= 2.3), git (>= 2.13)
```

Comment:
Any particular reason for 2.13 ?

Reply:
Yes, the Heroku deploy fails on earlier versions.